src/http/api/v1/exports.ts

```diff
@@ -84,13 +84,18 @@ export const createHandler = (config: HttpApiConfig): express.Router => {
       throw new Error(`Chain ${chainId} not configured`);
     }
 
+    const passportScoresByAddress =
+      await config.passportProvider.getScoresByAddresses(
+        votes.map((vote) => vote.voter)
```

Comment:
```suggestion
        votes.map((vote) => vote.voter.toLowerCase())
```

getVotesWithCoefficients expects addresses to be lowercase